students/sendecki_andrzej/lesson_06_dicts_tuples_sets_args_kwargs/lesson8-uppercase.py

```diff
@@ -0,0 +1,20 @@
+def capitalize(lower_case_word):
+    upper_case_letter = chr(ord(lower_case_word[0]) - 32)
+    upper_case_word = list(lower_case_word)
+    if 97 <= ord(lower_case_word[0]) <= 122:
```

Comment:
I know it's written along with instruction, however I would personally wrote is as:
`return word[0].upper() + word[1:]`

readability on the first place